data-raw/gostic_gt_pmf.R

```diff
@@ -0,0 +1,21 @@
+# E and I compartments both with exponentially distributed residence times
+# with a mean of 4 days.
+shape <- 2
+rate <- 1 / 4
+
+gostic_gt_pmf <- primarycensoreddist::dpcens(0:26,
+  pgamma,
+  shape = shape,
+  rate = rate,
+  D = 27
+) # v0.4.0
+
+# Drop first element because GI can't have same-day transmission
+gostic_gt_pmf <- gostic_gt_pmf[2:27]
+
+# Renormalize to a proper PMF
+while (abs(sum(gostic_gt_pmf) - 1) > 1e-10) {
+  gostic_gt_pmf <- gostic_gt_pmf / sum(gostic_gt_pmf)
```

Comment:
maybe open a EpiNow2 issue asking for this constraint to be relaxed? It just seems a bit painful!

Reply:
This is for here! We need to do a renormalization because of the left-truncation. 1e-10 is overkill but if we have to do it anyway, might as well be precise?